arc/benchmark.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# Arc ClickBench Complete Benchmark Script
+# This script installs Arc, loads data, and runs the benchmark
```

Comment:
I looked at all the files in this PR but I couldn't find a place where Arc is invoked. The benchmark runs using DuckDB as far as I see. Am I missing something?